src/cascadia/TerminalControl/TermControlAutomationPeer.h

```diff
@@ -49,6 +49,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         void UpdateControlBounds();
         void SetControlPadding(const Core::Padding padding);
         void RecordKeyEvent(const WORD vkey);
+        void IsClosing(bool closing);

```

Comment:
BTW it might be more readable if you name this function `NotifyControlClosed()` or something, since the boolean parameter will always be `true`.

Reply:
agreed. Or just `Close()` like the one in TermControl

Reply:
Yeah I like `Close()`. And I just realized that you can actually drop the `bool _closing` member and instead just `.reset()` the `_termControl` member!